edx_sysadmin/settings/common.py

```diff
@@ -5,7 +5,8 @@
 
 def plugin_settings(settings):
     """Settings for the edx-sysadmin plugin."""
-
+    settings.SYSADMIN_GITHUB_WEBHOOK_KEY = None
+    settings.SYSADMIN_DEFAULT_BRANCH = None
```

Comment:
The `None` value is intentional. Instead of setting a default value e.g. `master` or `releease` i preferred throwing an error when the `SYSADMIN_DEFAULT_BRANCH ` value isn't configured properly so that we don't accidentally keep on loading the courses from a branch that wasn't meant for a specif deployment.